### DISCUSSION TITLE
Don't work Configs کارنکردن کانفیگ ها

### DISCUSSION THREAD

**Discussion by AliAshooriyoon** [hiddify/Hiddify-Manager#3893] [Q&A سوالات رایج]
با درود  و خسته نباشید خدمت تیم خوب هدیفای 
بنده برا اولین بار هست که از این پنل استفاده میکنم ولی گویا به مشکلی برخوردم و هر چه قدر هم که جست و جو کردم راه حلی براش پیدا نکردم

مشکل من اینه که هر کاربری که تعریف میکنم هیچ کدوم از کانفیگ ها کار نمیکنه و timeout میده با اینکه از طریق ssh به سرور متصل شدم و مشکلی نداشت و فیلتر نشده بود و  همچنین ساب دامین هم پینگ گرفتم کاملا اوکی بود. 
تنظیمات پنل و همه مراحل رو طبق آموزش  چنل یوتویوبی به نام Iman Aghapur جلو رفتم.  
البته این رو هم ذکر کنم که این مشکل در روز اول وجود نداشت ولی امروز متوجه این مشکل شدم و هر چقدر هم که جست و جو کردم و راه حل های مختلف رو تست کردم موفق نشدم. ممنون میشم بنده رو راهنمایی کنید 

با سپاس

**Comment by yamorgonesh** [comment]
سلام اون روزهای اول درسته باز میشد اما به دلیل هجوم همه به این کار باعث شد به محض کپی شدن ، لاگ فیلتر میشه و از کار میکند افته و هر چه تلاش کنید بد و بدتر میشه
